drivers/sensor/sbs_gauge/sbs_gauge.h

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2021 Leica Geosystems AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_SBS_GAUGE_H_
+#define ZEPHYR_DRIVERS_SENSOR_SBS_GAUGE_H_
+
+#include <stdint.h>
+
+/*** Standard Commands ***/
+#define SBS_GAUGE_CMD_MANUFACTURER_ACCESS 0x00 /* ManufacturerAccess() */
```

Comment:
Can you please align it or place comments above a macro? Please remove () at the end.

```c
#define SBS_GAUGE_CMD_MANUFACTURER_ACCESS    0x00U /* ManufacturerAccess */
#define SBS_GAUGE_CMD_REM_CAPACITY_ALARM     0x01U /* LowCapacityAlarmThreshold */
```

Reply:
Values aligned and () removed at the end.